doc/rtd/development/datasource_creation.rst

```diff
@@ -1,7 +1,88 @@
 .. _datasource_creation:
 
-Datasource creation
-*******************
+Supporting your cloud or platform
+*********************************
+
+The upstream cloud-init project regularly accepts code contributions for new
+platforms that wish to support cloud-init.
+
+Ways to add platform support
+============================
+
+In order to use cloud-init with a new platform, a couple of approaches
+are possible:
```

Comment:
This sounds a bit passive. I'd say "To add cloud-init support for a new platform, there are two possible approaches".